docs/configure.md

```diff
@@ -52,13 +52,13 @@ Configuration steps in this file depend on [jq](https://stedolan.github.io/jq/),
 
 - [Configure gitserver replica count](#configure-gitserver-replica-count)
 - [Assign resource-hungry pods to larger nodes](#assign-resource-hungry-pods-to-larger-nodes)
-- [Configure Prometheus](../configure/prometheus/README.md)
-  - [Configure Alertmanager](../configure/prometheus/alertmanager/README.md)
+- [Configure Alertmanager](../configure/prometheus/alertmanager/README.md)
 - [Configure Jaeger tracing](../configure/jaeger/README.md)
 - [Configure Lightstep tracing](#configure-lightstep-tracing)
 - [Configure custom Redis](#configure-custom-redis)
 - [Configure custom PostgreSQL](#configure-custom-redis)
 - [Install without RBAC](#install-without-rbac)
+- [Use namespace](#use-namespace)
```

Comment:
```suggestion
- [Use non-default namespace](#use-non-default-namespace)
```

You'll need to check to see if the hyperlink still works.

Reply:
done